modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/describer/AnnotationsBasedDescriber.java

```diff
@@ -443,6 +446,15 @@ private void declareParameterGroups(Class<?> annotatedType, ParameterizedDeclare
                                                                           annotatedType));
             }
 
+            if (!isInstantiable(field.getType(), false))
```

Comment:
this is in the wrong place. This should be done in a ModelEnricher which uses an ExtensionWalker to verify all the parameters which have the proper model property. If you do it here, then you're not validating alternative describers which might have the same bug